src/scripts/components/LoginPage.tsx

```diff
@@ -133,29 +131,112 @@ function HeaderLinks ({ loggedIn }: Pick<LoginState, 'loggedIn'>) {
+            <Link href="settings.html"><Button>Settings</Button></Link>
+            <Link href="https://toggl.com/app" target="_blank"><p className="small">*To see older time entries, or run more complex reports, check out our web app!</p></Link>
+          </div>
+          <img src="../images/button-create.gif" />
+        </div>
+        <div css={[section, left, gold]}>
+          <div>
+            <h5>Supercharge your tools</h5>
+            <h3>Use integrations</h3>
+            <p>Add a Toggl Button to 130+ services for super fast time tracking!</p>
+            <p>Once enabled, just click on the Toggl Button icon to start a time entry. Toggl Button will automatically fill in the description for you.</p>
+            <p>If Toggl Button can detect a project, <em>and</em> there's project with the same name in Toggl, it'll be selected too!</p>
+            <p>You can stop entries by clicking the same icon again, or move straight onto another task by clicking the new task's icon.</p>
+            <p>Check out the Integrations page now to see if your tool is supported!</p>
+            <Link href="settings.html?tab=integrations"><Button>Integrations</Button></Link>
+          </div>
+          <img src="../images/button-integrations.gif" />
+        </div>
+        <div css={[section, right, green]}>
+          <div>
+            <h5>Increase your productivity</h5>
+            <h3>Pomodoro mode</h3>
+            <p>Need help managing your time?</p>
+            <p>Pomodoro is a technique to help you focus on a task, by working hard for short intervals then taking a break.</p>
+            <p>Toggl Button's pomodoro mode lets you set your interval length, and reminds you to take a break when time's up. You can also enable a ticking sound to help keep you on track.</p>
+            <p>Turn it on from the Pomodoro page.</p>
+            <Link href="settings.html?tab=pomodoro"><Button>Pomodoro</Button></Link>
+          </div>
+          <img src="../images/button-pomodoro.gif" />
+        </div>
+      </Guide>
+      <Spotty>
+        <h2>100% Open Source</h2>
+        <p>Want to help us make Toggl Button better?</p>
+        <Link href="https://github.com/toggl/toggl-button/" target="_blank">
+          <p>All our code is available on GitHub and we ❤ PRs!</p>
+          <svg height="32" viewBox="0 0 16 16">
+            <path fill-rule="evenodd" d="M8 0C3.58 0 0 3.58 0 8c0 3.54 2.29 6.53 5.47 7.59.4.07.55-.17.55-.38 0-.19-.01-.82-.01-1.49-2.01.37-2.53-.49-2.69-.94-.09-.23-.48-.94-.82-1.13-.28-.15-.68-.52-.01-.53.63-.01 1.08.58 1.23.82.72 1.21 1.87.87 2.33.66.07-.52.28-.87.51-1.07-1.78-.2-3.64-.89-3.64-3.95 0-.87.31-1.59.82-2.15-.08-.2-.36-1.02.08-2.12 0 0 .67-.21 2.2.82.64-.18 1.32-.27 2-.27.68 0 1.36.09 2 .27 1.53-1.04 2.2-.82 2.2-.82.44 1.1.16 1.92.08 2.12.51.56.82 1.27.82 2.15 0 3.07-1.87 3.75-3.65 3.95.29.25.54.73.54 1.48 0 1.07-.01 1.93-.01 2.2 0 .21.15.46.55.38A8.013 8.013 0 0016 8c0-4.42-3.58-8-8-8z"/>>
```

Comment:
![image](https://user-images.githubusercontent.com/1716853/69830331-73543600-124a-11ea-92c4-72b28741bbc0.png)

You might want to run it through https://github.com/smooth-code/svgr

Reply:
thanks - removed the attribute because it's not even necessary